diffusion_policy/common/sampler.py

```diff
@@ -98,6 +98,7 @@ def __init__(self,
         if episode_mask is None:
             episode_mask = np.ones(episode_ends.shape, dtype=bool)
 
+        sequence_length = sequence_length + 7
```

Comment:
This is fine for now, but we should consider generalizing this for the future before code release